docs/source/sparse.rst

```diff
@@ -53,10 +53,10 @@ __ https://en.wikipedia.org/wiki/Sparse_matrix
 Sparse COO tensors
 ++++++++++++++++++
 
-Currently, PyTorch implements the so-called Coordinate format, or COO
-format, as the default sparse storage format for storing sparse
-tensors.  In COO format, the specified elements are stored as tuples
-of element indices and the corresponding values. In particular,
+The default storage format for sparse tensors in PyTorch is the so-called
```

Comment:
What makes COO "the *default* storage format"? Although, it is implemented first but does there exist any place where the `sparse` (read: `sparse_coo`) layout is set as the default layout?

Reply:
Hmmm you have a point. Removed it.